pandas/io/formats/excel.py

```diff
@@ -95,8 +108,14 @@ def build_xlstyle(self, props):
 
         # TODO: handle cell width and height: needs support in pandas.io.excel
 
-        def remove_none(d):
-            """Remove key where value is None, through nested dicts"""
+        def remove_none(d: Dict[str, Any]) -> None:
```

Comment:
```suggestion
        def remove_none(d: Dict[str, Dict[str, str]]) -> None:
```

Minor nit but this should just match what `out` directly above is